container_pipeline/workers/base.py

```diff
@@ -29,16 +29,17 @@ def notify(self, data):
         """
         self.queue.put(json.dumps(data), 'master_tube')
 
-    def export_logs(self, logs, destination):
+    def export_logs(self, logs, destination, json_logs=False):
```

Comment:
I don't understand the problem here. I want to use one method to either export text or JSON logs. Text logs could be the service logs while JSON logs will be module execution status export and scanner results, etc. What is the problem if same method can export text as well as JSON. If one doesn't want to use `json_logs` variable, leave it, the default is set is to `False`. 

@dharmit WDYT ?